server/src/main/java/org/apache/kafka/security/CredentialProvider.java

```diff
@@ -19,11 +19,9 @@
 import org.apache.kafka.common.security.authenticator.CredentialCache;
 import org.apache.kafka.common.security.scram.ScramCredential;
 import org.apache.kafka.common.security.scram.internals.ScramCredentialUtils;
-import org.apache.kafka.common.security.scram.internals.ScramMechanism;
```

Comment:
Removing the import also fix the import conflict caused by `org.apache.kafka.clients.admin.ScramMechanism`. Hence, could you please import `org.apache.kafka.clients.admin.ScramMechanism` and then streamline line#36 and line#45?

Reply:
Sure!
I have just fixed it.